### ISSUE TITLE
Converted mp4 vanishes and not found in captured directory

### ISSUE THREAD

**Issue by jrudess** [jrudess/streamdvr#11]
There appears to be some race condition that appears to kill the ffmpeg post-process job which was converting ts -> mp4.   Then the ts is deleted, and the mp4 conversion did not finish, so the video is lost.

```
[20180128_130557] MFC      A stream is off.
[20180128_130557] MFC      [DEBUG] A is no longer broadcasting, ending ffmpeg process.
[20180128_130604] MFC      A converting to A_20180128_122640.mp4
[20180128_130604] MFC      [DEBUG] A is no longer broadcasting, ending ffmpeg process.
[20180128_130604] MFC      A done converting A_20180128_122640.mp4
```

**Comment by jrudess** [comment]
If you are hitting this, the temporary workaround is to disable post-process conversions in config.yml:

```
autoConvertType: mp4
```

**Comment by jrudess** [comment]
Fixed in v0.7.7
